packages/spruce-next-helpers/src/skillskit/helpers/lang.ts

```diff
@@ -0,0 +1,115 @@
+import * as moment from 'moment'
```

Comment:
Could this move to `spruce-utils`? This is perhaps more generic than just `next`